### DISCUSSION TITLE
Enable TODO highlighting for all filetypes

### DISCUSSION THREAD

**Discussion by craigmac** [nvim-treesitter/nvim-treesitter#4717] [Q&A] [answered]
How do I permanently enable TODO/FIXME/etc. highlighting in all filetypes, like Vim does by default? Not doing this breaks some of the shipped default themes like `quiet` when `set bg=light` because of TODO inside a comment only being linked to Comment by default instead of being marked as `@text.todo` which would link to `:hi Todo`. At least that's my limited understanding perspective on it. I've read `vim.treesitter` but I don't quite understand it enough from just that document alone. 

<img width="736" alt="Screen Shot 2023-04-26 at 2 35 13 PM" src="https://user-images.githubusercontent.com/7416158/234670990-2bdef20a-f862-412d-8cb9-64a0d45c5cd7.png">

How Todo is defined:

<img width="1182" alt="Screen Shot 2023-04-26 at 2 38 12 PM" src="https://user-images.githubusercontent.com/7416158/234671466-5a7f7d6c-5e42-4eca-812e-bd25df6db630.png">

I've tried putting in `~/.config/nvim/after/query/comments/highlights.scm` a few different things, but I have no idea what I'm doing. Any pointers?

I'm on macOS with nvim compiled from HEAD a few hours ago, after doing distclean and removing everything.

**Comment by ObserverOfTime** [comment, marked as answer]
The parser is named "comment", not "comments". Check if it's installed.

**Reply by craigmac** [reply]
Turns out it was LSP highlights having higher priority and making TODO be highlighted as simply Comment.
